roles/openshift_common/library/openshift_manage_iptables.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/python
```

Comment:
I went ahead and added the modeline to both the openshift_common/library/openshift_manage_iptables.py module and the openshift_node/library/openshift_register_node.py module.